src/tracks/Pattern.cpp

```diff
@@ -678,7 +678,7 @@ void PatternView::constructContextMenu( QMenu * _cm )
 
 	if( gui->pianoRoll()->currentPattern() &&
 			gui->pianoRoll()->currentPattern() != m_pat &&
-			not m_pat->empty() )
+			!m_pat->empty() )
```

Comment:
I think now it's time to revert this.

Reply:
I'll rebase on master, then it should be gone in this PR

Reply:
Okay.